src/components/Table/index.js

```diff
@@ -5,26 +5,30 @@ import glamorous from 'glamorous';
 const TableContainer = glamorous('table', {
   // use `forwardProps` here as by default glamorous doesn't allow setting `name` prop on a `table`
   forwardProps: ['name'],
-})({
-  display: 'table',
-  width: '100%',
-}, ({flexibleColumns}) => (
-  { tableLayout: flexibleColumns ? 'auto' : 'fixed' }
-));
+})(
+  {
+    fontFamily: 'nta, Arial, sans-serif',
```

Comment:
Just a note to say this is different from the font-stack applied on previous files in this PR.